README.md

```diff
@@ -171,4 +171,4 @@ Any use of third-party trademarks or logos are subject to those third-party's po
 
 <!-- Docs -->
 [MicrosoftAzureDocs]: <https://docs.microsoft.com/en-us/azure/>
-[PowerShellDocs]: <https://docs.microsoft.com/en-us/powershell/>
+[PowerShellDocs]: <https://docs.microsoft.com/en-us/powershell/>
```

Comment:
Was that intended?

Reply:
Unintended, restored the file to remove any changes.